Tests/scripts/secrets.py

```diff
@@ -19,7 +19,7 @@
 ENTROPY_THRESHOLD = 3.8
 
 SKIPPED_FILES = {'secrets_white_list', 'id_set.json', 'conf.json'}
-ACCEPTED_FILE_STATUSES = ['M', 'A']
+ACCEPTED_FILE_STATUSES = ['M', 'A', "R"]
```

Comment:
I'm not sure I like searching every R file for secrets is the right thing to do.
We have tried getting official docs for the many R types there are without success, I'd rather have an array of R's per case we encounter and verify to fit our flow over treating all R types the same way in advance.